src/platforms/botframework/BotFrameworkEvent.ts

```diff
@@ -64,6 +73,11 @@ export class BotFrameworkEvent extends VoxaEvent {
     this.initUser();
   }
 
+  public async getUserInformation(): Promise<any> {
```

Comment:
not sure what's this method? :thinking:

Reply:
I made this method abstract in the parent class VoxaEvent:

`public abstract async getUserInformation(): Promise<IVoxaUserProfile>;`

So, I put it in all the children classes. For `BotFrameworkEvent` is just a placeholder, that's why I left it blank